app/views/katello/api/v2/subscription_facet/subscription_status.json.rabl

```diff
@@ -0,0 +1,3 @@
+
+attributes :to_global => :color_code
```

Comment:
do you have a better proposal for indicating color for the UI? just the color strings?  'red' and 'green' ?